spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/rest/config/AdminConfiguration.java

```diff
@@ -60,6 +64,11 @@
 public class AdminConfiguration {
 
 	@Bean
+	public MetricRepository metricRepository(RedisConnectionFactory redisConnectionFactory) {
```

Comment:
I think we may need some conditions like `@ConditionalOnMissingBean(MetricRepository.class)` here.